tls-client/main.cpp

```diff
@@ -407,13 +429,14 @@ class HelloHTTPS {
 protected:
     TCPSocket* _tcpsocket;
 
-    const char *_domain;            /**< The domain name of the HTTPS server */
-    const uint16_t _port;           /**< The HTTPS server port */
-    char _buffer[RECV_BUFFER_SIZE]; /**< The response buffer */
-    size_t _bpos;                   /**< The current offset in the response buffer */
-    volatile bool _got200;          /**< Status flag for HTTPS 200 */
-    volatile bool _gothello;        /**< Status flag for finding the test string */
-    volatile bool _error;           /**< Status flag for an error */
+    const char *_domain;              /**< The domain name of the HTTPS server */
+    const uint16_t _port;             /**< The HTTPS server port */
+    char _recv_buf[RECV_BUFFER_SIZE]; /**< The response buffer */
+    char _crt_buf[CRT_BUFFER_SIZE];   /**< The server certificate buffer */
```

Comment:
Can you change the comment? It's misleading. This is a buffer for error text, not to contain 'server certificates', which could be misunderstood from the comment.